app/api/aact2/v1/app_api.rb

```diff
@@ -0,0 +1,44 @@
+module AACT2
+  module V1
+    class AppAPI < Grape::API
+      desc 'app status' do
+        detail 'this returns a health status'
+        named 'app_storage'
+        failure [
```

Comment:
Not a big deal, but it's kinda weird that the failure clause would include the 200 status. I'm sure that's just a Grape/Swagger idiosyncrasy. Nothing to change though.